src/Stratis.Bitcoin.Features.Consensus/Rules/CommonRules/BitcoinHeaderVersionRule.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using NBitcoin;
+using Stratis.Bitcoin.Base.Deployments;
+using Stratis.Bitcoin.Consensus.Rules;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.Consensus.Rules.CommonRules
+{
+    /// <summary>
+    /// Checks if <see cref="BitcoinMain"/> network block's header has a valid block version.
```

Comment:
How about bitcoin test net?

Reply:
I could remove the cref and simply refer to Bitcoin. Note that the existing `StratisHeaderVersionRule` and `BitcoinActivationRule` both refer to their respective main networks like this, so it is at least consistent to use it here.

Reply:
Reference to BitcoinMain is not resolved. Need to add
`using Stratis.Bitcoin.Networks;`

Reply:
Done